app/models/book.rb

```diff
@@ -93,6 +93,7 @@ class Book < ActiveRecord::Base
   settings number_of_shards: 1 do
     mapping do
       indexes "genre.name", index: 'not_analyzed'
+      indexes "subcategory.name", index: 'not_analyzed'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.